### ISSUE TITLE
[Works] DS423+ / DSM 7.2-64570 Update 1 / ASUS USB-C2500

### ISSUE THREAD

**Issue by jyubonk** [bb-qq/r8152#334]
Working well.

**Comment by jyubonk** [comment]
![image](https://github.com/bb-qq/r8152/assets/76234190/c18c5998-0206-4b46-9ab0-90ea7fc604ab)

![image](https://github.com/bb-qq/r8152/assets/76234190/8b329202-9955-4fd2-a59e-ccfb426dc67f)

**Comment by belag** [comment]
Yeah, works for me too. So far so good. Included is a local speedtest between my Macbook Air M2 with a Wavlink 2.5G USB adapter and the DS423+ with the Asus USB-C2500... :)
<img width="1437" alt="Screenshot 2023-08-24 at 4 20 07 PM" src="https://github.com/bb-qq/r8152/assets/29230564/5f1b0452-5a27-45dd-8d09-a127926c58a4">
